machine_data_model/nodes/connectors/opcua_connector.py

```
@@ -0,0 +1,440 @@
+            )
+        else:
+            validator = CertificateValidator(
+                CertificateValidatorOptions.EXT_VALIDATION
+                | CertificateValidatorOptions.PEER_SERVER
+            )
+        client.certificate_validator = validator
+
+        self._client = client
+        try:
+            await self._client.connect()
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)